plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/internal/UIConnectionMessages.properties

```diff
@@ -80,6 +80,7 @@ dialog_driver_manager_message_cant_delete_title = Cannot delete driver
 dialog_driver_manager_message_delete_driver_text = Are you sure you want to delete driver '
 dialog_driver_manager_message_delete_driver_title = Delete driver
 dialog_driver_manager_title = Driver Manager
+dialog_driver_manager_preferences_link = Global options can be changed in <a>{0}</a>
```

Comment:
Why do you call it options if it's a preference? As I recall, we usually don't use "options" for that